storage/innobase/btr/btr0cur.cc

```diff
@@ -5751,7 +5751,6 @@ ha_rows btr_estimate_n_rows_in_range(dict_index_t *index,
 /***********************************************************//**
 Gets the offset of the pointer to the externally stored part of a field.
 @return offset of the pointer to the externally stored part */
-static
 ulint
 btr_rec_get_field_ref_offs(
```

Comment:
Can we define this function inline in `btr0cur.h`:
```c++
inline size_t btr_rec_get_field_rec_offs(const rec_offs *offsets, size_t n)
  noexcept
{
  ut_ad(n > 0);
  ut_ad(n < rec_offs_n_fields(offsets));
  const rec_offs *f= rec_offs_base(offsets) + n;
  ut_ad((f[1] & TYPE_MASK) == STORED_OFFPAGE);
  size_t len= (f[1] - f[0]) & ~TYPE_MASK;
  ut_ad(len >= BTR_EXTERN_FIELD_REF_SIZE);
  ut_ad(len < UNIV_SQL_DEFAULT);
  static_assert(UNIV_SQL_DEFAULT + 1 == UNIV_SQL_NULL, "");
  return f[1] - STORED_OFFPAGE + len - BTR_EXTERN_FIELD_REF_SIZE;
}
```
In this way, we will avoid non-inline calls for rather simple code, which the above suggestion is simplifying further.